docker/.env.default

```diff
@@ -2,45 +2,55 @@
 # Create a copy of this file called .env
 # More Docs @ https://kitware.github.io/dive/Deployment-Docker-Compose
 
-# Stack name, used for containers, volumes, networks, etc
+# Docker Compose Stack configuration
+#
 COMPOSE_PROJECT_NAME=dive
 
 # Web server configuration
-PUBLIC_DATA_PATH=./girder_data
```

Comment:
removal of this means that the default instructions for copying .env.default and running don't work properly because it is trying to mount `girder_data` instead of `./girder_data`

Found in docker-compose.yml as:
`source: ${PUBLIC_DATA_PATH:-girder_data}`
figure it's probably best to change this to ./girder_data